Modules/vc-module-export/VirtoCommerce.ExportModule.Tests/PricingModuleModelMocks/Price.cs

```diff
@@ -0,0 +1,48 @@
+using System;
```

Comment:
Namespace here is not related to the folder where the models are located - PricingModuleModelMocks.
I think it would be better to make one folder containing test and models, like ExportMetadata or smth.